payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsAdapter.kt

```diff
@@ -11,11 +11,12 @@ import com.stripe.android.R
 import com.stripe.android.databinding.LayoutPaymentsheetAddCardItemBinding
```

Comment:
I am in the midst of a review, and just need a little more time to finish reviewing  before it is approved.

Reply:
I don't get it